src/mir/index.md

```diff
@@ -283,3 +351,5 @@ See the const-eval WG's [docs on promotion](https://github.com/rust-lang/const-e
 [`ProjectionElem::Deref`]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/mir/enum.ProjectionElem.html#variant.Deref
 [`Rvalue`]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/mir/enum.Rvalue.html
 [`Operand`]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/mir/enum.Operand.html
+[`mir::Constant`]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/mir/struct.Constant.html
```

Comment:
It seems the link is invalid right now and I can't easily find the corresponding struct type the link should point to.
@RalfJung: Can you please address it?

Reply:
`mir::Constant` has been renamed to [this](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/mir/enum.Const.html).

Reply:
Great! I'm going to include it in rust-lang/rustc-dev-guide#2024.